packages/system/velero/Makefile

```
@@ -0,0 +1,11 @@
+export NAME=vertical-pod-autoscaler
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)